examples/android_local_test/WORKSPACE

```diff
@@ -0,0 +1,25 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+android_sdk_repository(name = "androidsdk")
+
+local_repository(
+    name = "rules_jvm_external",
+    path = "../../",
+)
+
+load("@rules_jvm_external//:defs.bzl", "maven_install")
+
+maven_install(
+    artifacts = [
+        "androidx.appcompat:appcompat:1.0.2",
+        "androidx.annotation:annotation:1.0.2",
+        "androidx.test.ext:junit:1.1.0",
+        "org.robolectric:robolectric:4.1",
+        "org.robolectric:android-all:8.1.0-robolectric-4611349",
```

Comment:
This is a use case I probably should have thought of earlier: while it is uncommon to test against all apis, it is not uncommon to test against a few apis with your robolectric tests (you can use annotations in your test code to indicate which apis to test against). I definitely don't think this needs to be addressed in this CL, but note to self: the robolectric use case needs to be examined. I wonder how painful it would be to create a separate "maven_install" rule for each api you want to test against. The genrule/py binary that creates robolectric-deps.properties would need to be updated but other than that I think it would work

Reply:
Yeah, this is one case where the artifact versioning pinning wouldn't work. I was thinking about a macro / attribute to "force" coursier to download additional artifacts.